### PR TITLE
CMake updates for SDL3

### PR DESCRIPTION
**Pull request by madebr**

### CODE REVIEW

File: Xcode/pkg-support/resources/CMake/sdl3_mixer-config.cmake [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+set(SDL3_mixer_FOUND                TRUE)
+
+set(SDL3MIXER_VENDORED              TRUE)
+
+set(SDL3MIXER_CMD                   FALSE)
+
+set(SDL3MIXER_FLAC_LIBFLAC          FALSE)
+set(SDL3MIXER_FLAC_DRFLAC           TRUE)
+
+set(SDL3MIXER_GME                   FALSE)
```
Comment (madebr) [inline Xcode/pkg-support/resources/CMake/sdl3_mixer-config.cmake:22]: I'm assuming here the SDL3_mixer framework won't ship with gme support.

Comment (slouken) [reply Xcode/pkg-support/resources/CMake/sdl3_mixer-config.cmake:22]: I haven't looked at that yet, but I can flip that on if it does.

File: VisualC/pkg-support/cmake/sdl3_mixer-config.cmake [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+set(SDL3_mixer_FOUND                TRUE)
+
+set(SDL3MIXER_VENDORED              TRUE)
+
+set(SDL3MIXER_CMD                   FALSE)
+
+set(SDL3MIXER_FLAC_LIBFLAC          FALSE)
+set(SDL3MIXER_FLAC_DRFLAC           TRUE)
+
+set(SDL3MIXER_GME                   FALSE)
```
Comment (madebr) [inline VisualC/pkg-support/cmake/sdl3_mixer-config.cmake:21]: Same here, assume the VC SDK does not have gme support.

Comment (slouken) [reply VisualC/pkg-support/cmake/sdl3_mixer-config.cmake:21]: No, it doesn't.

File: Xcode/pkg-support/resources/CMake/sdl3_mixer-config.cmake (hunk 1 above)
Comment (sezero) [reply Xcode/pkg-support/resources/CMake/sdl3_mixer-config.cmake:22]: SDL2 version doesn't enable it by default either due to GME being not a very common necessity

**Comment by slouken** [comment]
Thanks! :)

**Pull request merged**